packages/grafana-ui/src/components/Logs/LogRowContext.tsx

```diff
@@ -197,6 +197,19 @@ export const LogRowContext: React.FunctionComponent<LogRowContextProps> = ({
   onLoadMoreContext,
   hasMoreContextRows,
 }) => {
+  const handleEscKeyDown = (e: KeyboardEvent): void => {
```

Comment:
@dprokop I wonder if it makes sense to have this inside `ClickOutsideWrapper`. I feel like this should be consistent and that you should be able to close anything that is modal like with `esc` key.